kinder/ci/workflows/regular-tasks.yaml

```diff
@@ -0,0 +1,135 @@
+# IMPORTANT! this workflow is imported by regular-* workflows.
+version: 1
+summary: |
+  This workflow implements a sequence of tasks used test the proper functioning
+  of kubeadm version X with Kubernetes version X.
+vars:
+  # vars defines default values for variable used by tasks in this workflow;
+  # those values might be overridden when importing this files.
+  kubernetesVersion: v1.13.5
+  controlPlaneNodes: 3
+  workerNodes: 2
+  baseImage: kindest/base:v20190403-1ebf15f
+  image: kindest/node:test
+  clusterName: kinder-xony
```

Comment:
kinder-regular?

Reply:
updated, also added https://github.com/kubernetes-sigs/kind/blob/master/pkg/build/node/node.go#L46
as the base image (from latest kind master) let's see if it works.